app/controllers/concerns/billable_event_trackable.rb

```diff
@@ -53,4 +53,12 @@ def session_has_been_billed_flag_key
       "auth_counted_#{issuer}"
     end
   end
+
+  def first_visit_for_sp?
+    issuer = sp_session[:issuer]
+
+    # check if the user has visited this SP at either IAL1 or IAL2 in this
+    # session
+    !user_session["auth_counted_#{issuer}ial1"] && !user_session["auth_counted_#{issuer}"]
```

Comment:
```suggestion
    # check if the user has visited this SP at either IAL1 or IAL2 in this session
    !user_session["auth_counted_#{issuer}ial1"] && !user_session["auth_counted_#{issuer}"]
```
our line length limit is 100 so we can probably just move that comment back up

Reply:
Done!